example-workflows/cloud-run/README.md

```diff
@@ -1,21 +1,25 @@
-# Cloud Run - GitHub Actions
+# Cloud Run Workflow
 
-An example workflow that uses [GitHub Actions][actions] to deploy a
-[Hello World Node.js app](index.js) to [Cloud Run][cloud-run].
+An example workflow that uses the `setup-gcloud` action to deploy to [Cloud Run][cloud-run].
+
+_**Checkout the [`deploy-cloudrun` action](https://github.com/google-github-actions/deploy-cloudrun) and [example workflows](https://github.com/google-github-actions/deploy-cloudrun/README.md#example-workflows)
+for a specialized implementation.**_
 
 This code is intended to be an _example_. You will likely need to change or
 update values to match your setup.
 
 ## Workflow description
 
-For pushes to the `master` branch, this workflow will:
+For pushes to the `example` branch, this workflow will:
```

Comment:
For triggering don't we need to copy the workflow from `example-workflows/cloud-run/cloud-run.yml ` to `.github/workflows`?